index.js

```diff
@@ -1,88 +1,121 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1;
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter++;
+
+console.log(counter); 
+ // => 2
+ console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+const name = "Seth";
+console.log(name);
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
-
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red")
+
+const drinkWater = currentThirstLevel => {
+  console.log("Man I sure am thirsty");
+  currentThirstLevel -= 1;
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel;
+}
+
 
-// console.log(drinkWater(12))
+console.log(drinkWater(12))
 // // => "Man I sure am thirsty"
 // // => "Ahh that hits the spot"
 // // => 11
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+const sameSameButDifferent = (arg1, arg2) => {
+  if (arg1 === arg2){
+    return "same same"
+  } else if (arg1 == arg2){
+    return "same same (but different)"
+  } else {
+    return "different"
+  }
+}
 
```

Comment:
Nice! The key here was understanding the difference between `===` and `==`. You could also convert the string to a number with parseInt to avoid == :